source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,169 @@
+
+:**Idle timeout**: Users will automatically disconnected after a defined amount of network inactivity.
+  The "last activity time of each user" is retrived using the last byte sent by each device. 
+  (meaning last byte "uploaded" from the user perspective)
+:**Hard timeout**: Users will be automatically disconnected after a defined time period, whether or not they are active.
+:**Traffic quota**: Users will be automatically disconnected after exceeding a defined amount of combined upload and download transfer data.
+
+
+If one of these options is enabled, pfSense will check once per minute to find users that meet criteria for disconnection.
+The way pfSense works is that a scheduled task (cron) will run every minute, checking for users to disconnect.
+
+.. tip:: These options disconnect users but they do not prevent users from immediately logging in again.
+ To prevent users from logging in again, use a RADIUS server for authentication with appropriately configured limits. See :ref:`label-radius-server`
+
+Authentication methods
+----------------------
+
+Four authentication methods are avaliable on pfSense :
+
+:Don't authenticate users: All users will be redirected to the captive portal
+  login page but will get connected as soon as they click **Login**. this is useful for
+  displaying a welcome page, or forcing users to see general terms and conditions.
+:Use an authentication server from the user manager: Authentication will be made using a remote authentication server. Please see :ref:`label-user-manager` for details.
+:Use RADIUS MAC Authentication: This authentication method emulate 802.1X, allowing some MAC addresses to get automatically connected.
+:Use vouchers: Vouchers are one-time use portal acces code. Please see :doc:`captive portal vouchers <captive-portal-vouchers>` page for details.
+
+
+Vouchers can be used in parallel with any authentication method, allowing for example
+login/password authentication for employees, and voucher auth for others users.
+
+
+.. _label-user-manager:
+
+Use an authentication server from the user manager
+--------------------------------------------------
+
+It is possible to authenticate users against a server from the server manager, such as an LDAP or RADIUS server.
+Please check the user manager documentation for details on how to setup remote authentication servers in pfSense.
+
+Remote authentication server have to be setup first in the server manager before being able to use it for captive portal authentication.
+
+
+Special note about "unauthenticated" username
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using an authentication server, it is recommanded to block/reserve the username "unauthenticated", in order to prevent users to connect using this username.
+This can be done by registering a disabled account in the remote authentication server (recommanded when using LDAP) or by preventing this user to login (recommanded when using a RADIUS server).
```

Comment:
*`recommended` (x2)

Change `preventing this user to login` to `preventing this user from logging in`